examples/src/rescue_raps/custom_trace_table.rs

```diff
@@ -0,0 +1,496 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#![allow(unused)]
```

Comment:
I'd prefer to remove the unused functionality from this module. We may bring it in in the future, but for now, the simpler we can make the test (and the PR) the better. For example, I'm not sure we need to bother with trace fragments for this.